docs/data/toolpad/concepts/authentication.md

```diff
@@ -0,0 +1,87 @@
+# Authentication
+
+<p class="description">Restrict a Toolpad application to authenticated users only, under certain authentication providers.</p>
+
+You can configure a Toolpad application so that users have to sign in with specific authentication providers in order to access it.
+
+{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/concepts/authorization/sign-in-page.png", "alt": "Toolpad sign-in page", "caption": "Toolpad sign-in page", "indent": 1, "aspectRatio": 1 }}
+
+Authentication settings can be accessed through the **Authorization** option in the app editor header.
+
+{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/concepts/authorization/authentication-settings.png", "alt": "Authentication settings", "caption": "Authentication settings", "indent": 1, "aspectRatio": 6 }}
+
+## Authentication secret
```

Comment:
Do we need to add a note here that it's important to keep this secret safe and not share it with anyone?